scripts/flood_run.py

```diff
@@ -17,60 +19,38 @@
 DATA_INTERMEDIATE = os.path.join(BASE_PATH, 'intermediate')
 DATA_PROCESSED = os.path.join(BASE_PATH, 'processed')
 
-
 path = os.path.join(DATA_RAW, 'countries.csv')
-path_2 = os.path.join(DATA_PROCESSED, 'RWA', 'population.csv')
-
-
-pop_tif_loc = os.path.join(DATA_RAW, 'WorldPop', 'ppp_2020_1km_Aggregated.tif')
-#filename = 'inunriver_rcp8p5_0000HadGEM2-ES_2080_rp01000.tif'
-flood_folder = os.path.join(DATA_RAW, 'flood_hazard')
 
-flood_files = os.listdir(flood_folder)
+filename = 'inunriver_rcp8p5_0000HadGEM2-ES_2080_rp01000.tif'
+flood_tiff = os.path.join(DATA_RAW, 'flood_hazard', filename)
 
-for file in flood_files:
-    try:
-        flood_pop = FloodProcess(path, 'RWA', file, path_2)
-        f = flood_pop.process_flood_tiff()
-        shp = flood_pop.process_tif()
-        cgdf = flood_pop.pop_flood()
-        merging = flood_pop.flood_pop_merge()
-        intersection = flood_pop.intersect_layers()
+#flooding = FloodProcess(path, 'KEN', flood_tiff)
```

Comment:
If this is unnecessary now, you should generally remove old code before pushign to the main branch.